.github/workflows/stale.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - uses: actions/stale@5ebf00ea0e4c1561e9b43a292ed34424fb1d4578 # v6
```

Comment:
# v6 is not being updated, thus I removed it.

Reply:
Strange, it should be updated as per https://github.com/dependabot/dependabot-core/issues/4691

Maybe it only updates full tags? (v6.0.1 in this case). We should set it to v7.0.0 and see if next time it works.

Reply:
👍